code/game/objects/items/ego_weapons/non_abnormality/kcorp.dm

```diff
@@ -0,0 +1,32 @@
+/obj/item/ego_weapon/city/kcorp
+	name = "K corp baton"
+	desc = "A green baton used by K corp employees."
+	icon_state = "kbatong"
+	inhand_icon_state = "kbatong"
+	force = 22
+	damtype = RED_DAMAGE
+	armortype = RED_DAMAGE
+	attack_verb_continuous = list("bashes", "crushes")
+	attack_verb_simple = list("bash", "crush")
+
+//Slows you to half but has really good defenses
+/obj/item/ego_weapon/shield/kcorp
```

Comment:
This one is so far stupidly good. Like, 0 requirements and a massively forgiving window of 3 seconds.
The only other shields we have that have this timing are Giant, which I believe is a secret one from the Beanstalk and climbing it (which can nearly kill you very easily), Swan which reduces it's damage by 1/3rd and isn't even in the game, and Hearse which is a Limbus EGO, none of which are near the resistances this has.
This shield is massively forgiving and the slow down is genuinely not notable enough from what I've seen. A clerk being able to outrun indigo dusks while using this should make that clear.

Mind you, the closest competitor to this in terms of resistance is Daredevil and that has 1/6th the block duration. Remember, when a shield blocks the debuff is not applied, so a Block time of 3 seconds means they need to avoid taking damage for 3 entire seconds to be punished at all.

Reply:
I'll look into changing these numbers, but it was bugged up until 20 seconds ago